### ISSUE TITLE
Support autotext for multiple previous words

### ISSUE THREAD

**Issue by PanderMusubi** [AnySoftKeyboard/AnySoftKeyboard#1066]
Please, support autotext for multiple previous words. Note that this is not trivial and should be limited in order to prevent recursion.

Example for Dutch: "trein wagon" → "treinwagon" and  "goederen trein wagon" → "goederentreinwagon" and not "goederen trein wagon" → "goederen treinwagon". This has impact on the order or rules and the word preceding the word group about to get autocorrected should always be checked too to see if no other rules apply.

In short, this is far from trivial and possibly dangerous when configured or implemented incorrectly.

**Comment by PanderMusubi** [comment]
Need to retest if this is already supported or not after https://github.com/AnySoftKeyboard/AnySoftKeyboard/issues/1069 has been solved.
